app/src/main/java/com/example/taskyy/data/room_database/TaskyyDatabase.kt

```diff
@@ -16,14 +16,14 @@ abstract class TaskyyDatabase: RoomDatabase() {
 
         @Volatile
         private var INSTANCE: RoomDatabase? = null
-        fun getDatabase(context: Context): RoomDatabase{
+        fun getDatabase(context: Context): TaskyyDatabase{
             if(INSTANCE != null) {
-                return INSTANCE as RoomDatabase
+                return INSTANCE as TaskyyDatabase
             }
```

Comment:
If you change the type for INSTANCE to `TaskyyDatabase`, you don't need that type cast here :)